src/SnackSession.js

```diff
@@ -292,6 +302,99 @@ export default class SnackSession {
     return url;
   };
 
+  /**
+   * Generates a default app.json template for the snack.
+   * @returns {Object} the app.json template
+   * @function
+   */
+  generateAppJson = (): Object => {
+    const appJsonTemplate = {
+      expo: {
+        slug: this.slug,
+        name: this.name,
+        description: this.description,
+        privacy: 'unlisted',
+        sdkVersion: this.sdkVersion,
+        version: '1.0.0',
+        orientation: 'portrait',
+        primaryColor: '#cccccc',
+        icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+        loading: {
+          icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+          hideExponentText: false,
+        },
+        packagerOpts: {
+          assetExts: ['ttf', 'mp4', 'otf'],
+        },
+        ios: {
+          supportsTablet: true,
+        },
+        android: {
+          package: "org.example.test",
+          permissions: ["CAMERA"],
+          versionCode: 1,
```

Comment:
i believe this will need to be incremented to create a new APK for the same snack

Reply:
I don't think it will, but it does need to be updated if it's uploaded to the app store. Anyways this is just the example, I'll add a comment saying that.